wizard/WizardCreateDevice1.qml

```diff
@@ -0,0 +1,184 @@
+                    labelFontSize: 14 * scaleRatio
+                    placeholderText: "<major>:<minor>"
+                    placeholderFontSize: 16 * scaleRatio
+                    validator: RegExpValidator { regExp: /(\d+):(\d+)?$/ }
+                }
+            }
+
+            ColumnLayout {
+                spacing: 0
+
+                Layout.topMargin: 10 * scaleRatio
+                Layout.fillWidth: true
+
+                ColumnLayout{
+                    MoneroComponents.StandardDropdown {
+                        id: deviceNameDropdown
+                        dataModel: deviceNameModel
+                        Layout.fillWidth: true
+                        Layout.topMargin: 6 * scaleRatio
+                        releasedColor: "#363636"
+                        pressedColor: "#202020"
+                    }
+                }
+            }
+
+            TextArea {
+                id: errorMsg
+                text: qsTr("Error writing wallet from hardware device. Check application logs.") + translationManager.emptyString;
+                visible: errorMsg.text !== ""
+                Layout.fillWidth: true
+                font.family: MoneroComponents.Style.fontRegular.name
+                color: MoneroComponents.Style.errorColor
+                font.pixelSize: 16 * scaleRatio
+
+                selectionColor: MoneroComponents.Style.dimmedFontColor
+                selectedTextColor: MoneroComponents.Style.defaultFontColor
+
+                selectByMouse: true
+                wrapMode: Text.WordWrap
+                textMargin: 0
+                leftPadding: 0
+                topPadding: 0
+                bottomPadding: 0
+                readOnly: true
+            }
+
+            WizardNav {
+                progressEnabled: false
+                btnNext.enabled: walletInput.verify();
+                btnPrev.text: qsTr("Back to menu") + translationManager.emptyString
+                btnNext.text: qsTr("Create wallet") + translationManager.emptyString
+                onPrevClicked: {
+                    wizardStateView.state = "wizardHome";
+                }
+                onNextClicked: {
+                    wizardController.walletOptionsName = walletInput.walletName.text;
+                    wizardController.walletOptionsLocation = walletInput.walletLocation.text;
+                    wizardController.walletOptionsDeviceName = wizardCreateDevice1.deviceName;
+
+                    if(restoreHeight.text)
+                        wizardController.walletOptionsRestoreHeight = parseInt(restoreHeight.text);
+                    if(lookahead.text)
+                        wizardController.walletOptionsSubaddressLookahead = lookahead.text;
+
+                    var written = wizardController.createWalletFromDevice();
+                    if(written){
+                        wizardStateView.state = "wizardHome";
```

Comment:
```suggestion
                        wizardStateView.state = "wizardCreateWallet2";
```
Otherwise, wizard will not advance to next page.

Reply:
Oh nice, you tested Ledger. I have yet to. Ill make these changes for the time being. Thanks.